src/components/TopBar/TopBarMediaControls.vue

```diff
@@ -120,6 +116,14 @@
 				</NcActionButton>
 			</template>
 		</NcActions>
+		<NcButton v-else-if="!screenSharingButtonHidden"
+			v-tooltip="screenSharingButtonTooltip"
+			type="tertiary"
+			@click.stop="toggleScreenSharingMenu">
+			<template #icon>
+				<MonitorShare :size="20" fill-color="#ffffff" />
```

Comment:
```suggestion
				<MonitorShare :size="20" />
```

Pls check for all icons. should be always `#fff` on dark theme / error type